src/wazuh_db/wdb.h

```diff
@@ -1033,6 +1045,16 @@ int wdb_parse_global_set_agent_labels(wdb_t * wdb, char * input, char * output);
  */
 int wdb_parse_global_update_agent_keepalive(wdb_t * wdb, char * input, char * output);
 
+/**
+ * @brief Function to parse the update the agent connection status.
+ *
+ * @param [in] wdb The global struct database.
+ * @param [in] input String with the agent data in JSON format.
+ * @param [out] output Response of the query.
+ * @return 0 Success: response contains the value OK. -1 On error: invalid DB query syntax.
```

Comment:
This line it's dificult to read. Maybe in different lines?
@return 0 on success: response contains "ok".
             -1 on error: response contains "err" and an error description

Reply:
FIXED for all the methods with the same **@return** comment.